test/integration/long/execution-profile-tests.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+var assert = require('assert');
+var util = require('util');
+
+var helper = require('../../test-helper.js');
+var Client = require('../../../lib/client.js');
+var types = require('../../../lib/types');
+var utils = require('../../../lib/utils.js');
+var loadBalancing = require('../../../lib/policies/load-balancing.js');
+var DCAwareRoundRobinPolicy = loadBalancing.DCAwareRoundRobinPolicy;
+var WhiteListPolicy = loadBalancing.WhiteListPolicy;
+var ExecutionProfile = require('../../../lib/execution-profile.js').ExecutionProfile;
+
+describe('ProfileManager', function() {
```

Comment:
I created this as a long test because it creates 6 nodes, but it isn't too slow (takes about 30 seconds including time to create/teardown server).